### PR TITLE
Move `Gameplay/Multiplayer` to `Client/Interface/Multiplayer`

### PR DESCRIPTION
**Pull request by Soetch**
**Based on #7165 from @The-Last-Cookie.**
Moved from Gameplay to Client/Interface (#7165).

Updated `Multi_team_vs.jpg`, `Multi_tag_co-op.jpg` and `Multi_tag_team_vs.jpg` (#7165).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

SKIP_WIKILINK_CHECK SKIP_OUTDATED_CHECK

### CODE REVIEW

File: wiki/Tournaments/GTS/AGTS_2019/fr.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ tags:
   - AGTS 2019
   - AGTS
   - GTS
-outdated_translation: true
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/GTS/AGTS_2019/fr.md:6]: This should be done in a separate PR.

Comment (Soetch) [reply wiki/Tournaments/GTS/AGTS_2019/fr.md:6]: Rhaaaaaa, I didn't change my branch, correcting this real quick.

Comment (Soetch) [reply wiki/Tournaments/GTS/AGTS_2019/fr.md:6]: Okay, done now.

Comment (The-Last-Cookie) [reply wiki/Tournaments/GTS/AGTS_2019/fr.md:6]: Sorry if I expressed myself poorly but the change was meant to be reverted. In other words, readd the line, so that the file will not show up in `git diff`.

In case you are unsure of what to do, just wait for a maintainer to pick this up and they will take care of it.

Comment (Soetch) [reply wiki/Tournaments/GTS/AGTS_2019/fr.md:6]: Yeah, I'm not that sure about what I'm doing, so I won't try, risking to break the commit lol. Thanks for taking time for me, I'm kinda clumsy.

**Comment by TPGPL** [comment]
made some changes to the PR besides rebasing

mainly fixed the broken links and redirects

**Review by TPGPL — APPROVED** [review]
